docs/operator/qos_rse_config.md

```diff
@@ -0,0 +1,47 @@
+---
+id: qos_rse_config
+title: QoS RSE Configuration
+---
+
+## QoS RSE Configuration
+
+The following steps show how to configure an RSE to use the Rucio storage-managed QoS feature.
```

Comment:
I would add `storage-managed QoS feature (with two classes, disk-buffer and tape)`